src/web/app/components/panel-link/panel-link.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
```

Comment:
Does this need to be a separate component?

Reply:
I created a new component because it is similar to panel chevron. However, we don't have to create a new component if we don't need this panel link component in any other pages.

Reply:
Probably not necessary in this current state. Just add icon directly to help panel.

Reply:
> Probably not necessary in this current state. Just add icon directly to help panel

👍🏻